libsrc/gamate/vga.inc

```diff
@@ -0,0 +1,228 @@
+
+; VGA charset for the PC-Engine conio implementation
+
+        .byte   $00, $00, $00, $00, $00, $00, $00, $00
+        ; 1
```

Comment:
Please add black (aka blank) lines before the comments.

BTW: I guess libsrc/pce/vga.inc would benefit from the comments too.